lefthook.yml

```
@@ -0,0 +1,12 @@
+pre-commit:
+  jobs:
+    - run: bun x ultracite format
+      glob: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)